Project-1/README.md

```diff
@@ -13,111 +13,345 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+  
+  C = gp2¯
+  But this doesn't make clear that these 2 are cousins as opposed to siblings. Either that, or it's only talking about a single person, rather than 2 cousins.
+  
   (d)  O that says that x is an only child
+  
+   O = ∃parent_of¯.(≤1∃parent_of ⊓ ≥1∃parent_of)
+   
+   or
+   
+   O ≡ ¬∃p2¯
+   I'm not a fan of this, since it's also true of rocks.
+  
   (e)  T that says that x has exactly two brothers 
+  
+  T = ∃parent_of¯.(≥3∃parent_of ⊓ ≥2∃parent_of.Male ⊓ ≤2∃parent_of.Male)
+  This doesn't quite work, since it says there's no more than 2 brothers out of the three, not that the one sibling I'm talking about has exactly 2 brothers.
+  
+  or
+  
+  p3 (parent of at least 3) = ≥3∃parent_of
+  T = ∃p3¯.(≥2∃parent_of.Male ⊓ ≤2∃parent_of.Male)
+  This proposal has the same problem as the previous one.
+  
 ```
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+```
+Tex = {Brother = M∩∃parent_of¯.(≥2∃parent_of)			(Tex.1)
+	Aunt = ~M∩∃parent_of¯.(∃parent_of.(∃parent_of))		(Tex.2)
+	Brother ⊆ Male,						(Tex.3)
+	Aunt ⊆ ~Male}						(Tex.4)
+
+Aex = {Billy: Brother ⊓ Male		(Aex.1)
+	Bob: Brother ⊓ Male		(Aex.2)
+	Mary: Aunt			(Aex.3)
+	(Alex, Billy): parent_of	(Aex.4)
+	(Alex, Bob): parent_of}		(Aex.5)
+	
+ΔI = {w, x, y, z},
+Billy = w
+Bob = x
+Mary = y
+Alex = z
+Male = {w, x}
+Brother = {w, x}
+Aunt = {y}
+Parent_of = {(z,w), (z,x)}
+```
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+  ```
+  (a) ∃x∀y and ∀x∃y
+  
+  ∃x∀y means "there is an x such that, for all y...", while ∀x∃y means "for all x, there is a y such that...". For instance, if I wanted to say that there is one particular thing that everything loves, I would write: "∃x∀yL(y,x)", but if I wanted to say that everything loves something without implying they all love the same thing, I would write: "∀x∃yL(x,y)". 
+  
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  
+  ∃x∀y∃z means "there is an x such that, for all y, there is a z such that..." while ∀x∃y∀z means "for all x, there is a y such that, for all z..."
+  
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  
+  ∀x∃y∀z∃w means "for all x, there is a y such that, for all z, there is a w such that...", while ∃x∀y∃z∀w means "there is an x such that, for all y, there is a z such that, for all w..."
+  
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+
+	0------> 1
+       ^ |      | ^
+       |_|      |_|
+       
+       -I'm not sure how to draw a graph in this program. I'm trying to show object 0 being in a relation with itself and with 1, but 1 only being in the relation with itself.
```

Comment:
You should check out mermaid syntax for markdown: https://mermaid.js.org/config/Tutorials.html

Github uses markdown for text editing, and mermaid is a flavor of markdown that allows you to draw diagrams.